File: introduction/ITP1_1_C-Rectangle.cs
using System;

public class Program {
  public static void Main() {
    var ab = Console.ReadLine().Split();
    int a = Int32.Parse(ab[0]);
    int b = Int32.Parse(ab[1]);
    Console.WriteLine(a*b + " " + (a+b)*2);
  }
}

File: introduction/ITP1_1_C-Rectangle.rs
use std::io;
fn main() {
  let mut input = String::new();
  io::stdin().read_line(&mut input).ok().expect("error");
  let ab: Vec<i32> = input.trim().split(" ").map(|s| s.parse().unwrap()).collect();
  println!("{0} {1}", ab[0]*ab[1], (ab[0]+ab[1])*2);
}
